Dockerfile

```diff
@@ -9,5 +9,3 @@ WORKDIR /app
 COPY requirements.txt /app/
 RUN pip install -r requirements.txt --no-cache-dir
 RUN apt-get update && apt-get install -y gettext libgettextpo-dev && apt-get clean
-
-COPY ./bookwyrm ./celerywyrm /app/
```

Comment:
Deleted because Flower _had_ been the only container that wasn't setting up the working directory as `/app` and now it's actively using the Celery config so it seemed to make sense to have it use the same volume that's live updated as the Celery workers